config/locales/fr.yml

```diff
@@ -25,8 +25,11 @@ fr:
       toggle_navigation: "Basculer navigation"
       toggle_sidebar: "Basculer vue latérale"
 
+    dialog:
+      error: "La requête n'a pas pu être traitée."
+
     confirmation:
-      title: "Voulez-vous supprimer?"
+      title: "Voulez-vous supprimer ?"
```

Comment:
Thanks for the PR. Just checking if the space before the `?` was intentional? I'm guessing it's a typo but I know very little French.

Reply:
Yes, exactly :) 
Most of special characters like `?`, `!`, `:`, `;`, etc. have a space before and after them; but not the dot. And yes, it doesn't apply to periods. Looks like you're not that bad in French 😄